README.md

```diff
@@ -96,6 +98,79 @@ As of version 0.2.0, you may also swipe a view programmatically:
 [self.swipeToChooseView mdc_swipe:MDCSwipeDirectionLeft];
 ```
 
+#### Swfit
```

Comment:
Small typo here, also you have to add `MDCSwipeToChooseDelegate` in the class declaration